.github/CODEOWNERS

```diff
@@ -394,6 +393,7 @@ x-pack/plugins/encrypted_saved_objects @elastic/kibana-security
 x-pack/plugins/enterprise_search @elastic/search-kibana
 x-pack/packages/kbn-entities-schema @elastic/obs-entities
 x-pack/plugins/observability_solution/entity_manager @elastic/obs-entities
+x-pack/plugins/observability_solution/entities_data_access @elastic/obs-entities
```

Comment:
You also need to change the owner in the `x-pack/plugins/observability_solution/entities_data_access/kibana.jsonc` to `@elastic/obs-entities`

Reply:
Fixed in https://github.com/elastic/kibana/pull/188537/commits/a4ed8f8dbabf378c75eb9047fce014b1d11a8186